src/models/generated/SwitchedAdmittance.jl

```diff
@@ -10,8 +10,9 @@ This file is auto-generated. Do not edit.
         available::Bool
         bus::ACBus
         Y::Complex{Float64}
-        number_of_steps::Int
-        Y_increase::Complex{Float64}
+        number_of_steps::Union{Int, Vector{Int}}
```

Comment:
Why do we need both `Int` and  `Vector{Int}`?

Reply:
I think I originally added this to enable contingencies x time steps. We can bring it to Vector Int.

Reply:
I think that PSSe has both options